src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -403,6 +420,17 @@ pub struct Args {
     /// The PostgreSQL URL for the storage stash.
     #[clap(long, env = "STORAGE_STASH_URL", value_name = "POSTGRES_URL")]
     storage_stash_url: String,
+    /// The Persist PubSub service URL.
+    ///
+    /// This URL is passed along with the port specified in `internal_persist_pubsub_listen_addr`
+    /// to `clusterd` for external discovery of the Persist PubSub service hosted within
+    /// `environmentd`.
+    #[clap(
```

Comment:
I'd strongly prefer if `ExternalHost:Port` were passed in to `environmentd` and I made a pass at it yesterday, but I ran into something tricky with the process orchestrator that I've already paged out somehow. I also think there's a chance I was misunderstanding some code, so lemme take a quick peek at this one again

Reply:
yeah I misunderstood things yesterday, replacing with `persist-pubsub-url` which is effectively `ExternalHost:Port`, and keeping `internal_persist_pubsub_listen_addr` which is going to be `0.0.0.0:Port` or `127.0.0.1:Port`

The ports don't actually need to match, I think it might be nice if our Kube Service maps port 80 to 6879 so all `clusterd` needs to know is `http://persist-pubsub` and they're off and running

Reply:
\o/ yeah that sounds much better